docs/analyzers/6.1/index.html

```diff
@@ -353,19 +353,19 @@ <h2 id="diagnostics">Diagnostics</h2>
 <td>FakeItEasy0003</td>
 <td>Argument constraint outside call spec</td>
 <td>no</td>
-<td>Triggered when you try to use an <a href="https://fakeiteasy.readthedocs.io/en/latest/argument-constraints/">argument constraint</a> outside of a <a href="https://fakeiteasy.readthedocs.io/en/latest/specifying-a-call-to-configure/">call specification</a>.</td>
+<td>Triggered when you try to use an <a href="https://fakeiteasy.github.io/docs/stable/argument-constraints/">argument constraint</a> outside of a <a href="https://fakeiteasy.github.io/docs/stable/specifying-a-call-to-configure/">call specification</a>.</td>
```

Comment:
```suggestion
<td>Triggered when you try to use an <a href="/docs/stable/argument-constraints/">argument constraint</a> outside of a <a href="/docs/stable/specifying-a-call-to-configure/">call specification</a>.</td>
```

?

Reply:
Oh! of course

Reply:
But maybe there is no point, since if we build and run locally /docs won't make much sense. As you say, this change is worthwhile in /index, but probably just there.